azext_edge/edge/providers/check/common.py

```diff
@@ -126,6 +133,8 @@ class DataLakeConnectorTargetType(ListableEnum):
 AIO_MQ_AUTH_PREFIX = "aio-mq-dmqtt-authentication"
 AIO_MQ_KAFKA_CONFIG_PREFIX = "aio-mq-kafka-config"
```

Comment:
do we still need this prefix (especially since we removed the KAFKA_CONNECTOR enum?) Check other prefixes too.